docker-compose.yml

```diff
@@ -97,6 +97,8 @@ services:
   zammad-railsserver:
     <<: *zammad-service
     command: ["zammad-railsserver"]
+    labels:
+      io.portainer.commands.bash-via-entrypoint: "/docker-entrypoint.sh /bin/bash"
```

Comment:
This looks good @YetAnotherGerrit. I haven't been able to test it yet, as my portainer extension in docker desktop yields errors when accessing the console feature. I may have to deploy portainer manually.

However, I would ask why you add this label here instead of to the `zammad-service` definition above, so that it can be used for all Zammad containers? That would seem more consistent and useful to me.

Reply:
I updated it as suggested.

As a note: this is not yet supported in Portainer LTS, it requires STS.
https://www.portainer.io/blog/2024-release-principle

Reply:
But this actually not due to the io.portainer.commands support, but as `lts` has a bug with the latest docker-ce regarding the "Console"-command as a whole. This is fixed with `sts`.